core/src/test/java/apoc/load/LoadJsonTest.java

```diff
@@ -259,46 +260,35 @@ public void testLoadJsonWithPathOptions() throws Exception {
 
     @Test public void testLoadJsonZipByUrl() throws Exception {
         URL url = new URL("https://github.com/neo4j-contrib/neo4j-apoc-procedures/blob/3.4/src/test/resources/testload.zip?raw=true");
-        testCall(db, "CALL apoc.load.json($url)",map("url",url.toString()+"!person.json"),
-                (row) -> {
-                    Map<String,Object> r = (Map<String, Object>) row.get("value");
-                    assertEquals("Michael", r.get("name"));
-                    assertEquals(41L, r.get("age"));
-                    assertEquals(asList("Selina", "Rana", "Selma"), r.get("children"));
-                });
+        assertEventuallySilently(db, "CALL apoc.load.json($url)", map("url", url +"!person.json"),
```

Comment:
Is this going to swallow any error that happens inside the CALL command?

Reply:
I changed the `assertEventuallySilently` in order to fails only with `SocketTimeoutException` class